src/lephare/data_manager.py

```diff
@@ -75,3 +72,26 @@ def configure_directories(self):
                 f"""User defined LEPHAREWORK is set. All intermediate files will
                 be written to {self.lephare_work_dir}."""
             )
+
+    # To be deprecated by pooch. For now just get all auxilliary data.
+    @classmethod
+    def get_auxiliary_data(cls, lephare_dir):
+        """Get all auxiliary data required to run lephare.
+
+        Function to be deprected by lephare internal pooch based retriever.
+
+        This gets all the filters, seds, and other data files.
+
+        Parameters
+        ==========
+        lephare_dir : `str`
+            The path to the lephare directory for auxiliary files.
+        """
+        # Assume if filt is present assume everything is.
+        if os.path.isdir(f"{lephare_dir}/filt"):
```

Comment:
Hmm, this might not be a sufficient check - when we create the `<this_os_cache>/data` directory in the `configure_directories` method above, we also create the subdirectories like `<this_os_cache>/data/filt`.

Reply:
One approach might be something like: 
```suggestion
        if os.listdir(f"{lephare_dir}/filt") != []:
```